package.json

```diff
@@ -91,9 +91,11 @@
     "multiselect": "~0.9.12",
     "number_helpers": "^0.1.1",
     "patternfly": "^3.58.0",
-    "patternfly-react": "^2.30.6",
+    "patternfly-react": "^2.33.0",
```

Comment:
I think you need to package this one too? at the time of writing its 2.32.4  ? (not seeing 2.33?)